numba/cuda/tests/cudadrv/test_nvvm_driver.py

```diff
@@ -25,21 +25,22 @@ def get_nvvmir(self):
 
     def test_nvvm_compile_simple(self):
         nvvmir = self.get_nvvmir()
-        ptx = llvm_to_ptx(nvvmir).decode('utf8')
+        ptx = nvvm.llvm_to_ptx(nvvmir).decode('utf8')
         self.assertTrue('simple' in ptx)
         self.assertTrue('ave' in ptx)
 
     def test_nvvm_from_llvm(self):
         m = ir.Module("test_nvvm_from_llvm")
         m.triple = 'nvptx64-nvidia-cuda'
+        nvvm.add_ir_version(m)
```

Comment:
Is this 3 line "setup" for a CUDA-target-LLVM-module sufficiently common to make it worth creating a utility function? The main concern here is that the `m.triple` specification is likely always necessary whereas the `nvvm.add_ir_version(m)` appears not to be? I suppose the underlying question is "what is the need driving the addition of the `add_ir_version()`" call?